<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QMainWindow" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>670</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="login">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>IP:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Port:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLineEdit" name="e_address"/>
                </item>
                <item>
                 <widget class="QSpinBox" name="sb_port"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="b_login">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chat">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTextEdit" name="te_input">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1920</width>
              <height>1080</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="b_selectFile">
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Select a file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="b_send">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="te_chat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="files">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QListView" name="lv_showFile"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_fileName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="b_backChat">
            <property name="text">
             <string>Go back to the chat </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="b_addFile">
            <property name="text">
             <string>Add file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
